.github/workflows/ci.yaml

```
@@ -402,17 +561,52 @@ jobs:
+            TAG="${ENV}-${TAG}"
+          fi
+
+          IMAGE="taiduc113/tastopia-${SERVICE}:${TAG}"
           sed -i "s|taiduc113/tastopia-${SERVICE}:.*|${IMAGE}|g" "$VALUES_FILE"
         done
       env:
         PR_TITLE: ${{ github.event.pull_request.title }}
         PR_URL: ${{ github.event.pull_request.html_url }}
     - name: Commit & push
+      id: commit_push
       run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:7:44: Double quote to prevent globbing and word splitting

(shellcheck)